conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -173,13 +173,18 @@ types:
           auth: optional<AuthType>
           args: list<ArgumentDefinition>
           returns: optional<Type>
+          errors: list<EndpointError>
           docs: optional<Documentation>
           deprecated: optional<Documentation>
           markers: list<Type>
           tags: set<string>
       EndpointName:
         alias: string
         docs: Should be in lowerCamelCase.
+      EndpointError:
+        fields:
+          errorName: TypeName
```

Comment:
If we match the pattern of `FieldDefinition`, I think perhaps this should be `error` rather than `errorName`, as it's effectively a reference to the error

Reply:
Makes sense: it is a reference to an error defined elsewhere.